src/chevah-python-tests/test_python_binary_dist.py

```diff
@@ -34,12 +34,13 @@ def get_allowed_deps():
         elif 'linux' in CHEVAH_OS:
             # Deps without paths for generic glibc Linux builds.
             # Only glibc 2.x libs are allowed.
-            # Tested on Amazon 2 & Ubuntu 16.04/18.04 with glibc 2.26/2.23/2.27.
+            # Tested on Amazon 2 & Ubuntu 20.04/22.04 with glibc 2.26/2.31/2.35.
             allowed_deps=[
                 'libc.so.6',
                 'libcrypt.so.1',
                 'libdl.so.2',
                 'libm.so.6',
+                'libnsl.so.1',
```

Comment:
Is this library directly needed by Python 3.12?

Can we build python without NIS module

https://docs.python.org/3/library/nis.html

This module is not used by the Chevah repos ... and will be removed soon

Reply:
Didn't bother much about it, because it's so small and unimportant, soon to be removed.

We used to patch `setup.py` for disabling such modules, but that file is gone. Other ideas of how to do this with 3.12?

Reply:
Fixed through https://github.com/chevah/pythia/pull/58/files/2303a256768d84384699427d6498fe9fb580eb2e..ac6595fba8c420b6f6a5a63559d7398cc29e4a6f.